### PR TITLE
spec: add os/linux/selinux-context isolator

### PR DESCRIPTION
**Pull request by lucab**
This commit introduces an "os/linux/selinux-context" isolator, which
can be applied at both app and pod scope.

### CODE REVIEW

**Comment by lucab** [comment]
@jonboulle @squeed PTAL.

File: spec/ace.md [hunk 1]
```diff
@@ -200,6 +200,37 @@ In the example above, the process will not be allowed to invoke `clock_adjtime(2
...
+
+**Parameters:**
+
+* **user** case-sensitive string containing the user portion of the SELinux security context to be used to label the current pod or application.
+* **role** case-sensitive string containing the role portion of the SELinux security context to be used to label the current pod or application.
+* **type** case-sensitive string containing the type/domain portion of the SELinux security context to be used to label the current pod or application.
+* **level** case-sensitive string containing the level portion of the SELinux security context to be used to label the current pod or application.
+
+**Notes:**
+ 1. Only a single `os/linux/selinux-context` isolator can be specified per-pod.
```
Comment (jonboulle) [inline spec/ace.md]: If a context is applied to a pod, does this just mean it's for all apps (unless they have overrides), or also for any executor processes involved in running the pod? (i.e. stage1 for rkt)

**Comment by jonboulle** [comment]
LGTM modulo clarification on pod thing

File: spec/ace.md (hunk 1 above)
Comment (lucab) [inline spec/ace.md]: It also applies to other processes in the pod context. Thanks for highlighting this, I'll make it explicit in the spec.

**Comment by lucab** [comment]
Amended with the clarification regarding pod-context.

**Comment by squeed** [comment]
:+1:

**Pull request merged**